app/configurations/config.matka.js

```diff
@@ -256,6 +257,15 @@ export default {
           en: 'https://kaupunkipyorat.kuopio.fi/?lang=2',
         },
       },
+      freebike_lahti: {
+        enabled: LahtiConfig.cityBike.networks.freebike_lahti.enabled,
```

Comment:
Nämä on näköjään tehty aina vähän hassusti. Muuta tämä ja kaikki muutkin citybike networkit näin:

freebike_lahti: lahtiConfig.cityBike.networks.freebike_lahti,
donkey_lappeenranta: LappeenrantaConfig.cityBike.networks.donkey_lappeenranta,
...